.github/workflows/README.md

```diff
@@ -0,0 +1,28 @@
+## Workflows Naming Convention
+
+Whenever possible, follow this naming convention for workflows:
+
+```yaml
+# Use a short descriptive name for the whole workflow
+name: Validate PR Labels
```

Comment:
I'd add that it should work nicely with the job names.

When it's displayed in the list of checks, it'd be nice for it to look something like:

`Validate / PR Labels`
`Validate / Localizations`
...etc

Reply:
Nice. Yeah, that would work better for workflows with more than one step that is not setup, like in your example.

In such case then it would also make sense to have an actual `name` for a job. My suggestion is to omit it, but then it would do

`validate / PR Labels`
`validate / Localizations`
...etc

That doesn't look as nice...